cves/CVE-2013-4310.yml

```diff
@@ -205,4 +219,6 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    Security should have been addressed more in the design of mapping actions,
```

Comment:
What can be done to prevent similar vulnerabilities in the future? Why do you think this is a design issue?

Reply:
I updated my answer to addressed your questions